scripts/win32.bat

```diff
@@ -54,5 +56,14 @@ For Each objDrive In colDiskDrives
             Wscript.Echo ""
         Next
     Next
+    If foundLDs = 0 Then
+      Wscript.Echo "device: """ & Replace(objDrive.DeviceID, "\", "\\") & """"
+      Wscript.Echo "description: """ & objDrive.Caption & """"
+      Wscript.Echo "size: " & objDrive.Size
+      Wscript.Echo "mountpoint: Null"
+      Wscript.Echo "name: Null"
+      Wscript.Echo "system: False"
```

Comment:
Hm, that's a tricky one. As it is, though, the only drive marked as system is the one the OS is running on. Other non-removable drives are still designated as `system: false`.

Another option would be to set it as `null` I guess?